docs/prerequisites.md

```diff
@@ -0,0 +1,27 @@
+#Prerequisites
+
+##Windows
+
+Ensure you have the following installed:  
+Must Have:  
+[Node.js x64](https://nodejs.org/en/)  
+[Python 2.7](https://www.python.org/downloads/)  
+[Visual Studio 2013 or 2015](https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx)  
+
+Should Have:
+
+[Git](https://git-scm.com/download/win)
+
+##OS X
```

Comment:
Please only list Windows for now since there is no information below that is useful.